### DISCUSSION TITLE
C02 emissions for 2023 changing daily

### DISCUSSION THREAD

**Discussion by jokersten** [electricitymaps/electricitymaps-contrib#6398] [General]
Hello, specific CO2 emissions for Germany (production) for the year 2023 are today at 424 specific CO₂-emissions gCO₂eq/kWh. Yesterday it was at 416, one week ago at 441. Why are those numbers still changing daily for the "finshed" year 2023? 

Thanx and best regards
Jo

**Comment by VIKTORVAV99** [comment]
Hi!

Would it be possible to get some more info here?
Maybe a screenshot?
These should not be changing daily, in fact we don't even compute these figures daily.

**Reply by jokersten** [reply]
Jan. 04: 443g
Jan. 12: 441g
Jan. 26: 421g

Screenshots attached

Jan. 22: 416g (no screenshot)


![Screenshot 24 01 04](https://github.com/electricitymaps/electricitymaps-contrib/assets/157689773/36a24ac7-8f0e-44c4-aca7-cd1daddc0d68)
![Screenshot 24 01 12](https://github.com/electricitymaps/electricitymaps-contrib/assets/157689773/fa18e836-3133-47e2-b9c8-40a59ff41471)
![Screenshot 24 01 26](https://github.com/electricitymaps/electricitymaps-contrib/assets/157689773/3ba0dda2-1325-4264-915c-b6982bd1132a)

**Reply by VIKTORVAV99** [reply]
We recently reprocessed a lot of data for 2021 to 2023 which might have caused this. But it should not have changed more than once.

We continuously re-process data in order to ensure we have the most accurate values we possible can.

I'll keep an eye out for any other changes though.

**Reply by jokersten** [reply]
It changed more than once, as you can see in the screenshots.
